tools/apiDocs.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs')
+const path = require('path')
+const spec = require('../spec/api')
+const { OpenApi } = require('../src/openApiSDK')
+
+const docsTarget = path.join(__dirname, '../src/methods4docs.js')
+
+/**
+ * Generate dynamic methods into `docsTarget` file, to be consumed by jsDocs `generate-docs` task
+ * Generated file is just for documentation purposes
+ *
+ */
+const generateMethods = () => {
+  const apiConfig = OpenApi.__getApiConfig(spec)
+  console.log('Total methods: ', apiConfig.length)
+  return apiConfig.map(config => {
+    const params = config.parameters.map(param => {
+      const jsdocParam = `parameters.${param.name}`
```

Comment:
regarding the `types.d.ts` issue (in another comment), I believe if you single quote param.name it will solve it:
`const jsdocParam = `parameters.'${param.name}'`
(but this quotes everything, even if it does not need it)

Reply:
Quotes added only if name contains dot (params.'some.name', params.name)
Initially I've tried square brackets as a valid param jsDoc (params['some.name']), but jsdoc generator scripts messed up markdown if main param object is optional ([params['some.name']] - this is valid but doesn't work with jsdoc to md script), so this is a workaround.